backend/app/services/repositories/user_repository.py

```
@@ -0,0 +1,88 @@
+            password_hash=password_hash,
+            name=user_data.name,
+            role=user_data.role,
+            is_active=True,
+        )
+
+        self.db.add(user)
+
+        try:
+            self.db.commit()
+            self.db.refresh(user)
+            return user
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)